tests/nightly/gpu/test_convai2.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+import parlai.core.testing_utils as testing_utils
+
+
+@testing_utils.skipUnlessGPU
+class TestConvai2Seq2Seq(unittest.TestCase):
+    """
+    Checks that the Convai2 seq2seq model produces correct results.
```

Comment:
maybe some language in this file to indicate that these were competition baselines